css/syntaxes.json

```diff
@@ -368,6 +368,9 @@
   "keyframe-selector": {
     "syntax": "from | to | <percentage>"
   },
+  "layer()": {
+    "syntax": "layer( <layer-name> )"
+  },
```

Comment:
I couldn't find anywhere this is defined in a spec, can you point me to it? It looks as if this participates in the definition of `@import` (https://www.bram.us/2021/09/15/the-future-of-css-cascade-layers-css-at-layer/#cascade-layers--create), so maybe the syntax for `@import` should be updated to refer to it? But I can't find any reference to `layer()` in the spec for `@import`, or in our page.

Reply:
It's the function used to declare a layer within the `@import` rule https://drafts.csswg.org/css-cascade-5/#layer-declaration - I've updated the syntax for the this as well - and yes the docs need updating but I'll get to that in due course 👍